recipes/scallop-lr/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "scallop-lr" %}
+{% set version = "0.9.2" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/Kingsford-Group/scallop/releases/download/isoseq-v{{ version }}/scallop-lr-{{ version }}.tar.gz"
+  sha256: "30b9fbd56110416f7471857e359069e1ab4dc412015cf7d385fb80c0bc87170a"
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - htslib
+    - libboost
```

Comment:
can you please try to use `boost-cpp`

Reply:
something like this?
```  
  - {{ compiler('boost-cpp') }}
  host:
    - htslib
    - libboost
```

Reply:
I tried but failed: https://circleci.com/gh/bioconda/bioconda-recipes/69817

Could you direct me the exact configuration that I need to use? Thanks!

For now, I changed back to the previous one that passed the tests.

Reply:
Thanks!